### PR TITLE
Symfony cheatsheet

### PR DESCRIPTION
**Pull request by oxygensend**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [x] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [x] All your assets are stored in the **assets** folder.
- [x] All the images used are in the **PNG** format.
- [x] Any references to websites have been formatted as [TEXT](URL)
- [x] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [x] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #1222.

Thank you again for your contribution :smiley:

In the document I covered the most common vulnerabilities and presented good practices to increase the security  of Symfony apps listed in the issue. I decided to leave topics like API Rate Limiting, Encryption which are to general and not strict to Symfony. I also tried to make the scheme not to differ from cheat sheets from Ruby on Rails and Laravel.
This is my first CS and pr in opensource world, so feel free to comment,  I am open for any proposals of changes :)

### CODE REVIEW

**Review by edg84 — COMMENTED** [review]
[Symfony_Cheat_Sheet.md](https://github.com/OWASP/CheatSheetSeries/files/13460159/Symfony_Cheat_Sheet.md)

**Review by jmanico — DISMISSED** [review]
This is very solid work, I'd like this to go live soon.

File: cheatsheets/Symfony_Cheat_Sheet.md [hunk 1]
```diff
@@ -0,0 +1,578 @@
+# Symfony Cheat Sheet
+
+## Introduction
+
+This *Cheat Sheet* aims to provide developers with security tips when building applications using the Symfony framework.
```
Comment (szh) [inline cheatsheets/Symfony_Cheat_Sheet.md:5]:
```suggestion
This cheat sheet aims to provide developers with security tips when building applications using the Symfony framework.
```

**Review by szh — COMMENTED** [review]
I have no knowledge of Symfony so I won't review the content, but here's quick formatting nit.

**Pull request merged**